home/views.py

```diff
@@ -42,10 +42,16 @@ def new_question(request):
     if request.method == 'POST':
         questForm = QuestionForm(request.POST)
         if questForm.is_valid():
-            questForm = questForm.save(commit=False)
-            questForm.profile = request.profile
-            questForm.save()
-            return redirect('question-detail', questForm.id)
+            tags_array = [t.strip().lower() for t in questForm.cleaned_data['tags_'].split(',')]
+            tags_status = Tag.check_tag_array(tags_array)
+            if(not questForm.data['tags_'] or tags_status):
```

Comment:
Not seeing updates to the tests of this view that reflect these code changes, also, assuming `check_tag_array` returns false, how do reflect that fat to the end user?

Reply:
You can see the updates to the tests that reflect these code changes in :
1. `test_tag_model_functions.py` - There is a test there that checks the new model function of Tag, `check_tag_array`.
2. `test_insert_question_feature.py` - I changed the parameterize of `valid_data` and `invalid_data` with some data of tags, as you can see in that file.
As a result, the tests in methods `test_add_valid_question` and `test_add_invalid_question` also check the insertion of the tags. 

Regarding `check_tag_array`, I thought about putting a message display mechanism in the case that `check_tag_array` is `False` in a separate PR.
Do you think it should be in this PR? (Because of time constraints)

@ifireball